### PR TITLE
Fixes #7856: Use a mustache template from a string

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7856

### CODE REVIEW

File: tree/30_generic_methods/file_from_string_mustache.cf [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#####################################################################################
+# Copyright 2014 Normation SAS
```
Comment (jooooooon) [inline tree/30_generic_methods/file_from_string_mustache.cf]: 2016 :)

File: tree/30_generic_methods/file_from_string_mustache.cf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# @parameter destination     Destination file
+#
+# @class_prefix file_from_template
+# @class_parameter destination
+# This bundle will define a class file_from_template_${destination}_{kept,repaired,not_ok,ok,reached}
+
+bundle agent file_from_string_mustache(template, destination)
+{
+  vars:
+    "class_prefix"        string => canonify("file_from_template_${destination}");
```
Comment (jooooooon) [inline tree/30_generic_methods/file_from_string_mustache.cf]: Shouldn't this be file_from_string_mustache?

Comment (peckpeck) [inline tree/30_generic_methods/file_from_string_mustache.cf]: Maybe, i don't really know what it "should" be

Comment (jooooooon) [inline tree/30_generic_methods/file_from_string_mustache.cf]: Well unless you have a specific reason for it not to be, this should be the bundle name, so yeah, "file_from_string_mustache"

**Comment by peckpeck** [comment]
PR updated

**Comment by ncharles** [comment]
Why targeting v0.x for 3.7 only code ?
v0.X is used on Rudder 3.1, which uses 3.6 version for the agent

**Comment by jooooooon** [comment]
ncf is not only used by Rudder :-)

**Comment by ncharles** [comment]
who would use branch v0.x with a newer cfengine ?

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/ncf/pull/322

**Pull request closed without merging**